tests/test_reviewer.py

```diff
@@ -0,0 +1,59 @@
+from pathlib import Path
+from unittest.mock import MagicMock
+
+import pytest
+
+from app.reviewer import Reviewer
+from dtos import CodeReviewDto, CriteriaDto, DirectoryDto
+
+
+class TestReviewer:
+    @pytest.fixture
+    def mock_models_client(self):
+        return MagicMock()
+
+    @pytest.fixture
+    def mock_parser(self):
+        return MagicMock()
+
+    @pytest.fixture
+    def mock_prompt_builder(self):
+        return MagicMock()
+
+    @pytest.fixture
+    def reviewer(self, mock_models_client, mock_parser, mock_prompt_builder):
+        return Reviewer(
+            llm_api_client=mock_models_client,
+            parser=mock_parser,
+            prompt_builder=mock_prompt_builder
+        )
+
+    def test_review(self, reviewer, mock_parser, mock_prompt_builder, mock_models_client):
+        test_dir = Path("test_dir")
```

Comment:
if it is created 4real, you can create a temp dir for it: https://docs.pytest.org/en/6.2.x/tmpdir.html